python/lsst/ts/Dome/dome_csc.py

```diff
@@ -183,8 +188,8 @@ async def do_moveAz(self, _data):
 
         Parameters
         ----------
-        _data : `A SALOBJ data object`
-            Contains the data as defined in the SAL XML file.
+        _data : `A SALOBJ configuration_parameters object`
```

Comment:
I think this is a search/replace error. Also...why the leading underscore on the data argument?

By the way: the back ticks tell sphinx to try to make a link, so they are only appropriate if the type is an actual type.

Reply:
Thanks. PyCharm must have messed up due to some refactoring that I did.

The underscore prevents PyCharm from complaining when the parameter is not used. In cases where it IS used I could remove them. But perhaps for uniformity of the API I can leave them in place everywhere. I do prefer to keep the underscore since it is one warning less and I prefer to have my IDE without warnings.